apps/dashboard/src/components/subscribers/subscriber-list-empty.tsx

```diff
@@ -0,0 +1,31 @@
+import { AddSubscriberIllustration } from '@/components/icons/add-subscriber-illustration';
+import { RiBookMarkedLine } from 'react-icons/ri';
+import { Link } from 'react-router-dom';
+import { LinkButton } from '../primitives/button-link';
+
+export const SubscriberListEmpty = () => {
+  return (
+    <div className="flex h-full w-full flex-col items-center justify-center gap-6">
+      <AddSubscriberIllustration />
+      <div className="flex flex-col items-center gap-2 text-center">
+        <span className="text-foreground-900 block font-medium">No subscribers yet</span>
```

Comment:
```suggestion
        <span className="text-foreground-900 block font-medium text-label-md">No subscribers yet</span>
```

You can follow the existing typography classes we have which match this comment from figma. 
![CleanShot 2025-01-21 at 11 14 35@2x](https://github.com/user-attachments/assets/639e613d-26d7-4627-b725-a3e63068e305)